superset/assets/src/explore/components/AdhocFilterEditPopoverSimpleTabContent.jsx

```diff
@@ -163,6 +188,12 @@ export default class AdhocFilterEditPopoverSimpleTabContent extends React.Compon
     }
   }
 
+  multiComparatorRef(ref) {
+    if (ref) {
```

Comment:
are there cases where you don't get a `ref`? that seems strange but mostly curious.

Reply:
I don't understand why but I've seen refs come in as `undefined` sometimes